### PR TITLE
Add Pterodactyl

### PR DESCRIPTION
**Pull request by Celemo**
I will start a thread at the forum when the app will be ready to test.

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -2185,6 +2185,13 @@
         "state": "notworking",
         "url": "https://github.com/abeudin/proftpd_ynh"
     },
+    "pterodactyl": {
+        "branch": "master",
+        "revision": "HEAD",
+        "maintained": true,
+        "url": "https://github.com/aeris-studio/pterodactyl_ynh",
+        "state": "inprogress"
+    }
```
Comment (tituspijean) [inline apps.json]: You forgot a comma after the `}` ;)

**Comment by arcdigital** [comment]
Hey, before you merge this - I'm on the pterodactyl dev team. I'd like to get the existing app (https://github.com/YunoHost-Apps/pterodactyl_ynh) listed and officially maintained by us. Happy to have Celemo contribute since he's already started putting in some work on getting it updated - but would prefer to have the package maintained officially by us.

**Comment by alexAubin** [comment]
(As a side note folks, I see this app (like others) is using php 7.2. Please consider using 7.3 right away as it will smooth the transition to buster. Otherwise thanks for your work!)

File: apps.json (hunk 1 above)
Comment (tituspijean) [reply apps.json]: @arcdigital's comment is quite relevant actually. Why fork the repo already hosted in YunoHost-Apps, and not contribute your improvements back into it?

**Comment by Celemo** [comment]
I'm actually working on a full recode of the app based on example_ynh app. 

I'll send a pull request when it will be ready.
I already opened an issue on the Yunohost-Apps' Pterodactyl repo where I asked if I could become the new maintainer. The original maintainer has stopped all activity on the repo since the last year and the old package is still installing v0.7.13 instead of v0.7.17.

I'll edit this PR to include the YunoHost-Apps' repo instead of my fork.

**Comment by Celemo** [comment]
On my recode I'm using php7.3 ^^

**Comment by Celemo** [comment]
PR Edited

File: apps.json (hunk 1 above)
Comment (Celemo) [reply apps.json]: Comma added

**Comment by maniackcrudelis** [comment]
@Celemo @arcdigital you can share the maintenance of this app if you want.
Considering there's already old issues on this repo without answers, if you agree to share this app, I can quickly give you both an access to this repo.

**Comment by arcdigital** [comment]
@Celemo would you be okay with allowing me to be the maintainer and you can submit any changes via PR and I'll review & merge? I'd like to ensure we enforce a proper code review & test process, so the app remains both stable and secure.

**Comment by Celemo** [comment]
I totally agree, if @arcdigital agrees too.

**Comment by Celemo** [comment]
Maybe we can be both two maintainers ?
About the proper code review, a simpler thing would be to have two branches, master and testing.
Actually the app is broken, so well first need to have a working version.

**Comment by maniackcrudelis** [comment]
Especially if you want to use [`"revision": "HEAD"`](https://github.com/YunoHost/apps/pull/889/files#diff-f471a89ee1e3d366a26176737308c654R2190), you should have a testing branch.

**Comment by Celemo** [comment]
I just reread the conversation, seems like I misunderstood something.
When @arcdigital say :
"- but would prefer to have the package maintained officially by us.".
Maintained officially by the Pterodactyl team ? That would be great !

**Comment by arcdigital** [comment]
Yes, that's what I meant. I would love to work with you though, please submit a PR when you have your version working 🙂

**Comment by maniackcrudelis** [comment]
Is it ok for you now guys ?
Should we merge this ?

**Comment by Celemo** [comment]
@maniackcrudelis, sure, you can merge.
Can you also give repo access to @arcdigital ?

**Pull request merged**